File: Level1/예산.py
def solution(d, budget):
    answer = 0
    
    #오름차순 정렬
    d.sort()
    
    #예산이 크다면
    while budget < sum(d):
        #가장 큰 값 제거
        d.pop()
    
    #정답
    answer = len(d)
    
    #반환
    return answer

File: Level1/예산.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int sum(vector<int>& v) {
    int tot = 0;
    for (int i = 0; i < v.size(); ++i)
        tot += v[i];

    return tot;
}

int solution(vector<int> d, int budget) {
    int answer = 0;

    //오름차순 정렬
    sort(d.begin(), d.end());

    while (budget < sum(d))
        d.pop_back();

    answer = d.size();
    return answer;
}